usaspending_api/common/helpers/spark_helpers.py

```diff
@@ -22,6 +22,39 @@
 from usaspending_api.common.helpers.aws_helpers import is_aws, get_aws_credentials
 from usaspending_api.config import CONFIG
 
+from usaspending_api.accounts.models import FederalAccount, TreasuryAppropriationAccount
+from usaspending_api.recipient.models import StateData
+from usaspending_api.references.models import (
+    Cfda,
+    Agency,
+    ToptierAgency,
+    SubtierAgency,
+    NAICS,
+    Office,
+    PSC,
+    RefCountryCode,
+    CityCountyStateCode,
+    PopCounty,
+    PopCongressionalDistrict,
+)
+
+RDS_REF_TABLES = [
```

Comment:
I like the use of models, but are there any tables we'd consider "reference" ... or maybe just _internal_ tables that we don't have a model for? e.g. `external_data_load_date` 
... turns out, not that one. It has model `borker.ExternalDataLoadDate`

I think if it came to this, you could add some type-sensing. If str, use it, if model, get the table name from it.

You can resolve this comment after read.